doc/versions.json

```diff
@@ -13,9 +14,15 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.3.99",
-      "nrfxlib": "1.3.99",
-      "zephyr": "2.3.99",
+      "ncs": "1.4.99",
+      "nrfxlib": "1.4.99",
+      "zephyr": "2.4.99",
```

Comment:
Why is this 2.4.99? This should be 2.4.0

Reply:
Fixed

Reply:
Actually, 2.4.99 is more correct, since this is what will be displayed for the /latest/ folder. In the /v1.4.0/ folder, the data from the v1.4.0 entry is displayed.
It doesn't really make a difference if it is changed in this PR or in https://github.com/nrfconnect/sdk-nrf/pull/3240 though.